src/main_thread/decrypt/content_decryptor.ts

```diff
@@ -291,6 +297,22 @@ export default class ContentDecryptor extends EventEmitter<IContentDecryptorEven
           return;
         }
 
+        if (
+          shouldCallGenerateRequestBeforeBufferingMedia(mediaKeySystemAccess.keySystem)
+        ) {
+          try {
+            await performFakeGenerateRequest(mediaKeys);
```

Comment:
Note: maybe this does not need to be done if `MediaKeySessions` have already been created (and their `generateRequest` method called)?

To check, it could save us some time when playing multiple contents